### PR TITLE
Ticket 4 get api reviews reviewid

### PR DESCRIPTION
**Pull request by steve2116**

### CODE REVIEW

File: controllers/api.controller.js [hunk 1]
```diff
@@ -38,6 +38,42 @@ module.exports = (request, response) => {
                     },
                 },
             },
+            "/api/review/:review_id": {
+                get: {
+                    status: "OK",
+                    info: "A review object with a specified id",
+                    data: "The review requested through a parametric endpoint",
+                    keys: [
+                        "review_id",
```
Comment (pippaamy) [inline controllers/api.controller.js:47]: WOAH lets import this in here rather than pasting the whole json

**Review by pippaamy — CHANGES_REQUESTED** [review]
Good work, lets make these changes and pop in another pr :)

File: controllers/api.reviews.id.controller.js [hunk 2]
```diff
@@ -7,6 +7,17 @@ exports.getReviewById = (request, response, next) => {
             response.status(200).send({ review: review });
         })
         .catch((err) => {
-            next(err);
+            if (err.code && err.msg) {
```
Comment (JVal-NC) [inline controllers/api.reviews.id.controller.js:10]: This looks ok, but the logic should definitely be in an error handler function, or functions. Take a look at the error handling notes.

**Review by JVal-NC — COMMENTED** [review]
Stevie, have a look at error handling notes. logic looks ok but want to see it in error handling functions before a full review.

**Pull request merged**